legend-depot-core-services/src/main/java/org/finos/legend/depot/services/api/projects/ProjectsService.java

```diff
@@ -74,7 +74,9 @@ default ProjectDependencyReport getProjectDependencyReport(String groupId, Strin
         return getProjectDependencyReport(Arrays.asList(new ProjectVersion(groupId, artifactId, versionId)));
     }
 
-    List<ProjectDependencyWithPlatformVersions> getDependentProjects(String groupId, String artifactId, String versionId);
+    List<ProjectDependencyWithPlatformVersions> getDependantProjects(String groupId, String artifactId, String versionId);
```

Comment:
you don't need the service on line 77 anymore

Reply:
you can make it have a default implementation that uses the method belw so you can remove from the impl